library/core/src/ptr/mut_ptr.rs

```diff
@@ -2359,8 +2405,10 @@ impl<T: ?Sized> PartialOrd for *mut T {
 
 #[cfg(kani)]
 #[unstable(feature = "kani", issue = "none")]
-mod verify {
+pub mod verify {
```

Comment:
Curious why this is necessary?

Reply:
Please refer to the this [comment](https://github.com/model-checking/verify-rust-std/pull/188#discussion_r1862968896).